packages/rule-no-bom/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "author": "",
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "Warns against using the BOM character at the beginning of a text file",
```

Comment:
For consistency, use `sonarwhal rule that against using the BOM character at the beginning of a text file`.